files/ts/app/templates/application.gts

```diff
@@ -0,0 +1,18 @@
+import Route from 'ember-route-template';
```

Comment:
this is a bit of a divergence from the default template. I understand that people will be using ember-route-template when working with gts but this is very very different. I don't want this divergence in our app blueprint

Reply:
I _do_ want to add a `--compatibility` flag or something in a followup PR that adds _all the compat_

Reply:
lets not add this for now, we can discuss it later if we want to add it

Reply:
I don't really want to go back

Reply:
> but then realized that in that case it's actually just much better to build the same feature into ember.

winning all around